app/lib/account/backend.dart

```diff
@@ -398,6 +379,11 @@ class GoogleOauth2AuthProvider extends AuthProvider {
         return null;
       }
 
+      if (info.userId == null) {
```

Comment:
Just uncomment the lines above which say:
```
          // TODO: Enable when pub client is requesting the correct scopes.
          // info.userId == null ||
          // info.userId.isEmpty ||
```

Reply:
good catch!